### ISSUE TITLE
FTBFS: oc-react-components needs react-15 while groupfolders want react-17

### ISSUE THREAD

**Issue by StCyr** [nextcloud/groupfolders#1549]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. make clean
2. delete package-lock.json
3. make

### Expected behaviour
package should build

### Actual behaviour

```
cyrille@vm-cyrille-debian:/var/www/nextcloud/apps/groupfolders$ make
npm install --deps
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.2" from oc-react-components@0.2.0
npm ERR! node_modules/oc-react-components
npm ERR!   oc-react-components@"^0.2.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/cyrille/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/cyrille/.npm/_logs/2021-06-18T09_27_49_906Z-debug.log
make: *** [Makefile:24: node_modules] Error 1
```

### Server configuration

using npm 7.5.2 to build (though I don't think it's related)

**Comment by StCyr** [comment]
my bad. this only happens in my branch.

Rebasing it solved the issue

**Comment by StCyr** [comment]
too soon. It does happen on master too

**Comment by joshtrichards** [comment]
Fixed in #1624
